ovspeed_classes.map

```diff
@@ -0,0 +1,95 @@
+CLASS
```

Comment:
don't use map extension for includes, use .inc extension 
don't place the inc files in the root of the project, create an includes folder and place the *.inc files in a folder that matches the map name.
please correct this for speed_classes.map as wel